tests/_files/VariousIterableDataProviderTest.php

```diff
@@ -7,9 +7,9 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
-class VariousIterableDataProviderTest
+class VariousIterableDataProviderTest extends AbstractVariousIterableDataProviderTest
 {
-    public static function asArrayProvider()
+    public function asArrayProvider()
```

Comment:
I renamed them and wrote similar tests. I hope it is clear, would be happy to hear how it could be done better

Reply:
I'd generally suggest reverting the old asset changes: they do prevent regressions (like the one that you are fixing - and BTW thanks very much for your efforts!)

Reply:
I've now kept the old test, but named it `testWithVariousIterableStaticDataProviders` instead of `testWithVariousIterableDataProviders`, and then also made tests for when this provider is non-static, of the parent, called from the parent, etc.

Reply:
Thanks!